Added a command line installation section to the README. (#72)
schrodinger/maeparser: README.md

* Quantum Mechanics applications, such as Jaguar
* Protein-Protein Docking applications
* Many other backends used in both Life and Material Sciences

Installation
============

Command line installation on a
[Unix-like](https://en.wikipedia.org/wiki/Unix-like)
operating system follows a typical configure, build, and test procedure.
Configuration is via [CMake](https://en.wikipedia.org/wiki/CMake);

> building is via [Make](https://en.wikipedia.org/wiki/Make_(software));
> and testing can be performed by running a custom built program.  Here is an
> example command sequence:

Reviewer: CMake can be instructed to  generate the build instructions for different build generators (see the `-G` option in the CMake documentation: https://cmake.org/cmake/help/latest/manual/cmake.1.html). `make` is just the default option (e.g., `make` won't be available on windows systems).

```suggestion
and testing can be performed by running a custom built program.  Here is an
example command sequence building via [Make](https://en.wikipedia.org/wiki/Make_(software)) and the `gcc` compiler:
```

> ```bash
> git clone git@github.com:schrodinger/maeparser.git maeparser
> cd maeparser
> mkdir build
> cd build
> export CC=gcc
> export VERBOSE=yes
> cmake `pwd/..`
> make
> test/unittest 
> ```

Reviewer: ```suggestion
```bash
# "clone" a local copy of the code from github
$ git clone git@github.com:schrodinger/maeparser.git maeparser

# set up the build configuration
$ cd maeparser
$ mkdir build
$ cd build
$ export CC=gcc
$ export VERBOSE=yes
$ cmake `pwd/..`

# Run the build
$ make

# Run the tests
$ test/unittest 
```
```

> Defining CC ensures that the first
> [compiler](https://en.wikipedia.org/wiki/Compiler) in one's
> [PATH](https://en.wikipedia.org/wiki/PATH_(variable)) is used,
> and defining VERBOSE enables viewing the gory details of compiling and
> [linking](https://en.wikipedia.org/wiki/Linker_(computing))
> that will be necessary to reproduce when one builds one's own program
> that uses the maeparser library.

Reviewer: Some comments here:

- There might be other compilers installed, but setting CC requires one specific compiler, not necessarily in the PATH (`CC` can also be specified with an absolute path to the compiler, and in such case, it doesn't have to be in the PATH variable).

- The "gory details" are not required to use maeparser in one's code (the default output level is enough for that), but they are useful if there's trouble.

```suggestion
Defining CC ensures that the specified
[compiler](https://en.wikipedia.org/wiki/Compiler) is used in the build 
(in the example, it will be the first instance of `gcc` in one's [PATH](https://en.wikipedia.org/wiki/PATH_(variable))),
and defining VERBOSE enables viewing the gory details of compiling and
[linking](https://en.wikipedia.org/wiki/Linker_(computing))
that will be necessary for debugging or reporting issues if the build fails.
```
